tools/c7n_azure/c7n_azure/policy.py

```diff
@@ -86,41 +85,67 @@ class AzureFunctionMode(ServerlessExecutionMode):
 
     POLICY_METRICS = ('ResourceCount', 'ResourceTime', 'ActionTime')
 
-    default_storage_name = "cloudcustodian"
+    default_storage_name = "custodian"
 
     def __init__(self, policy):
 
         self.policy = policy
         self.log = logging.getLogger('custodian.azure.AzureFunctionMode')
-
         self.policy_name = self.policy.data['name'].replace(' ', '-').lower()
+        self.function_app_name = None
+
+    def configure(self):
```

Comment:
nit: not sure this is a good name.. get_function_app_params() or smth to reflect what it returns?

Reply:
Agreed